src/main/java/seedu/address/model/appointment/Appointment.java

```diff
@@ -24,56 +18,78 @@ public class Appointment {
      * @param description A description of the appointment.
      */
     public Appointment(DateTime dateTime, Person student, Description description) {
-        requireAllNonNull(dateTime, student, description);
         this.dateTime = dateTime;
         this.student = student;
         this.description = description;
     }
 
+    /**
+     * Retrieves the date and time of the appointment.
+     *
+     * @return The date and time of the appointment.
+     */
     public DateTime getDateTime() {
         return dateTime;
     }
 
+    /**
+     * Sets the date and time of the appointment.
+     *
+     * @param dateTime The new date and time for the appointment.
+     */
+    public void setDateTime(DateTime dateTime) {
+        this.dateTime = dateTime;
+    }
```

Comment:
Is there a need for a setter here ?

Reply:
I believe this might be needed in case of the future where we need to reschedule an appointment or a separate method can be created just for that purpose, we can discuss it when we are developing that feature